Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -496,21 +500,21 @@ private static DateTime setEndValidTime(DateTime endValidTimeArg, DateTime issue
     protected List<AttributeStatement> createEvidenceStatements(final CallbackProperties properties) {
         LOG.debug("SamlCallbackHandler.createEvidenceStatements() -- Begin");
 
-        final List accessConstentValues = properties.getEvidenceAccessConstent();
-        final List evidenceInstanceAccessConsentValues = properties.getEvidenceInstantAccessConsent();
+        final List accessConstentValues = checkHcidPrefixInList(properties.getEvidenceAccessConstent());
+        final List evidenceInstanceAccessConsentValues = checkHcidPrefixInList(properties.getEvidenceInstantAccessConsent());
 
         return createEvidenceStatements(accessConstentValues, evidenceInstanceAccessConsentValues);
     }
 
     public List<AttributeStatement> createEvidenceStatements(final List accessConstentValues,
         final List evidenceInstanceAccessConsentValues) {
         List<AttributeStatement> statements;
-        if (accessConstentValues == null) {
+        if (CollectionUtils.isEmpty(accessConstentValues)) {
```

Comment:
These 2 if conditions and log statements are unnecessary. We can rew3ord and write these log statement in OpenSaml2Builder.createEvidenceStatements method.